mkosi.conf.d/05-qemu-mem.conf

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=arch centos
```

Comment:
maybe a comment on why this file only applies to centos and arch would be useful